src/utilities/FormManager.ts

```diff
@@ -115,6 +115,22 @@ class FormManager {
     return formValues;
   }
 
+  public get submitModel (): IModel {
+    const submitModel: IModel = {};
+
+    this.fieldConfigs
+      .filter(fieldConfig => !filterInsertIf(fieldConfig, this.formModel))
```

Comment:
This dual filtering out of `readOnly` and false `insertIf` happens elsewhere, and will probably be built upon with other filters as `fields-ant` grows. We should think about a helper that "cleans" `fieldConfigs`, though right now it would probably be overkill

Reply:
Agreed.